packages/astroport-governance/src/generator_controller.rs

```diff
@@ -80,8 +87,12 @@ pub struct ConfigResponse {
     pub factory_addr: Addr,
     /// Max number of pools that can receive an ASTRO allocation
     pub pools_limit: u64,
-    /// Max number of blacklisted voters can be removed
+    /// Max number of blacklisted voters which can be removed
     pub blacklisted_voters_limit: Option<u32>,
+    /// Main pool that will receive a minimum amount of ASTRO emissions
+    pub main_pool: Option<Addr>,
```

Comment:
We should add this fields to InstantiateMsg

Reply:
No, we don't

Reply:
Why? We do not have a deployed contract yet or Is already there?

Reply:
There is no such requirement in the task